Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:latest
+
+MAINTAINER Justin Henderson justin@hasecuritysolutions.com
+
+RUN yum update -y
+RUN yum install -y python python-devel git gcc
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+RUN python get-pip.py
+
+WORKDIR /opt/VulnWhisperer
+
+COPY requirements.txt requirements.txt
+COPY setup.py setup.py
+COPY vulnwhisp/ vulnwhisp/
+COPY bin/ bin/
+COPY configs/frameworks_example.ini frameworks_example.ini
+
+RUN python setup.py clean --all
```

Comment:
Here we will be missing the `cd deps/qualysapi && python setup.py install && cd ../../` now that we are using again the custom fork from the VulnWhisperer repository, otherwise it will pull the pip qualysapi version, which was the one that had the issue with Qualys Web module.

Reply:
When I clone vulnwhisperer, I don't get the deps/qualysapi.  i could change it so that it downloads a specific version from GitHub.